etcdctl/ctlv3/command/del_command.go

```diff
@@ -70,6 +72,9 @@ func getDelOp(args []string) (string, []clientv3.OpOption) {
 			cobrautl.ExitWithError(cobrautl.ExitBadArgs, fmt.Errorf("too many arguments, only accept one argument when `--prefix` or `--from-key` is set"))
 		}
 		opts = append(opts, clientv3.WithRange(args[1]))
+		ignoreWarn = false
+	} else if delPrefix || delFromKey {
+		ignoreWarn = false
```

Comment:
It's not needed.... We don't need IMHO warning if user explicitly stated '--from-key' or `--prefix`.
This logic would print the warning in such situation as well.